### DISCUSSION TITLE
https://mokkapps.de/vue-tips/force-enable-vue-devtools-in-production-build

### DISCUSSION THREAD

**Discussion by giscus** [Mokkapps/mokkapps-de-discussions#8] [Announcements]
# https://mokkapps.de/vue-tips/force-enable-vue-devtools-in-production-build

Usually, the Vue Devtool extension is disabled in production builds, but there is a way to force-enable it.

https://mokkapps.de/vue-tips/force-enable-vue-devtools-in-production-build

**Comment by xinwo** [comment]
In my case, it's Vue2 and Vue Devtools extension doesn't show

**Comment by johnlwebb3** [comment]
For me, using Vue3, this gets the components to show but not the props and variables and such.

**Comment by kplaricos** [comment]
I would like to do the same for my Nuxt App. i have an issue with my Staging server but not in local. `const app = Array.from(document.querySelectorAll('*')).find((e) => e.__vue_app__).__vue_app__
` is not available in Nuxt i guess

**Reply by Mokkapps** [reply]
Yes, I do not know if this even possible with Nuxt 🤷🏻‍♂️
